### PR TITLE
feat(imageGallery): pinch-to-zoom implementation

### PR DESCRIPTION
**Pull request by kibo-chandradeeptalaha**

### CODE REVIEW

File: components/KiboImageGallery.vue [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+            class="thumbnail__stack--arrow"
+            @click="() => handleVerticalSlider(true)"
+          >
+            <SfChevron class="sf-chevron--top" />
+          </div>
+
+          <div id="scrollable" role="tablist" class="thumbnail__scrollable">
+            <div
+              v-for="(image, i) in images"
+              :key="i"
```
Comment (kibo-kevinwatts) [inline components/KiboImageGallery.vue]: dont use index for key

File: components/KiboImageGallery.vue [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+              :alt="selectedImage.image.alt"
+              width="100%"
+              height="100%"
+            />
+          </PinchScrollZoom>
+        </client-only>
+        <div class="thumbnail__dots-container">
+          <div
+            v-for="(image, i) in images"
+            :key="i"
```
Comment (kibo-kevinwatts) [inline components/KiboImageGallery.vue]: avoid using index for key

File: components/KiboImageGallery.vue (hunk 1 above)
Comment (kibo-chandradeeptalaha) [reply components/KiboImageGallery.vue]: Changed

File: components/KiboImageGallery.vue (hunk 2 above)
Comment (kibo-chandradeeptalaha) [reply components/KiboImageGallery.vue]: Changed

Comment (kibo-chandradeeptalaha) [reply components/KiboImageGallery.vue]: added imageUrl as key

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**